packages/odyssey/src/scss/abstracts/_colors.scss

```diff
@@ -55,6 +55,7 @@ $colors: (
     '500': #e97107
   ),
   'turquoise': (
+    '400': #65adb7,
```

Comment:
Do we want this to be deployed with Odyssey markup? If not, let's move this to the theme layer. If so, please add it to our [color](https://github.com/okta/odyssey/blob/master/packages/docs/base/color.md) file.